src/reactjs_jsx_ppx_v4.ml

```diff
@@ -0,0 +1,1455 @@
+("hrefLang", {jsString = None; typeString = mkPropConst (Lident "string")});
+("htmlFor", {jsString = None; typeString = mkPropConst (Lident "string")});
+("httpEquiv", {jsString = None; typeString = mkPropConst (Lident "string")});
+("icon", {jsString = None; typeString = mkPropConst (Lident "string")});
+("inputMode", {jsString = None; typeString = mkPropConst (Lident "string")});
+("integrity", {jsString = None; typeString = mkPropConst (Lident "string")});
+("keyType", {jsString = None; typeString = mkPropConst (Lident "string")});
+("kind", {jsString = None; typeString = mkPropConst (Lident "string")});
+("label", {jsString = None; typeString = mkPropConst (Lident "string")});
+("list", {jsString = None; typeString = mkPropConst (Lident "string")});
+("loop", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("low", {jsString = None; typeString = mkPropConst (Lident "int")});
+("manifest", {jsString = None; typeString = mkPropConst (Lident "string")});
+("max", {jsString = None; typeString = mkPropConst (Lident "string")});
+("maxLength", {jsString = None; typeString = mkPropConst (Lident "int")});
+("media", {jsString = None; typeString = mkPropConst (Lident "string")});
+("mediaGroup", {jsString = None; typeString = mkPropConst (Lident "string")});
+("method", {jsString = None; typeString = mkPropConst (Lident "string")});
+("min", {jsString = None; typeString = mkPropConst (Lident "string")});
+("minLength", {jsString = None; typeString = mkPropConst (Lident "int")});
+("multiple", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("muted", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("name", {jsString = None; typeString = mkPropConst (Lident "string")});
+("nonce", {jsString = None; typeString = mkPropConst (Lident "string")});
+("noValidate", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("open_", {jsString = Some("open"); typeString = mkPropConst (Lident "bool")});
+("optimum", {jsString = None; typeString = mkPropConst (Lident "int")});
+("pattern", {jsString = None; typeString = mkPropConst (Lident "string")});
+("placeholder", {jsString = None; typeString = mkPropConst (Lident "string")});
+("poster", {jsString = None; typeString = mkPropConst (Lident "string")});
+("preload", {jsString = None; typeString = mkPropConst (Lident "string")});
+("radioGroup", {jsString = None; typeString = mkPropConst (Lident "string")});
+("readOnly", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("rel", {jsString = None; typeString = mkPropConst (Lident "string")});
+("required", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("reversed", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("rows", {jsString = None; typeString = mkPropConst (Lident "int")});
+("rowSpan", {jsString = None; typeString = mkPropConst (Lident "int")});
+("sandbox", {jsString = None; typeString = mkPropConst (Lident "string")});
```

Comment:
According to [WHATWG HTML living standard](https://html.spec.whatwg.org/#attr-iframe-sandbox), this should be
> an unordered set of unique space-separated tokens

which means that, in theory, we should take a _pseudo-list_ of a variant type that can be transformed into a string. I don't know if this can be expressed, and it's not the end of the world if it cannot be done -- just worth mentioning it.